tinygrad/shape/symbolic.py

```diff
@@ -195,6 +195,10 @@ def get_bounds(self) -> Tuple[int, int]:
   def substitute(self, var_vals: Dict[VariableOrNum, Node]) -> Node: return self.a.substitute(var_vals) < (self.b if isinstance(self.b, int) else self.b.substitute(var_vals))
 
 class MulNode(OpNode):
+  def __lt__(self, b: Union[Node, int]):
+    if isinstance(b, Node) or isinstance(self.b, Node): return Node.__lt__(self, b)
```

Comment:
This line is odd: if `isinstance(b, Node)` false, it's not a Node, so how can it have a `.b` member? Mypy should trigger here.

Reply:
self is Node, but self.b or b can be int.

I did not understand the problem

Reply:
You're completely right. Sorry.